devtools/support/debug.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+echo "=====================================HEADER====================================="
+echo "Run this script with your conda env activated"
+echo "Invoke the script like this: "
+echo "./debug.sh | tee -a debug.log"
+echo "Then send us debug.log"
+
+set -euo pipefail
+
+echo -e "\n=====================================SYSTEM====================================="
+date
+
+echo "$OSTYPE" || echo "could not detect operating system"
+arch
+
+nvidia-smi || echo "no nvidia-smi, are you on a gpu node?"
```

Comment:
(not blocking) It's a little weird to phrase this as a question, I think it's better to output this as a statement. Users who can understand it will try to debug for themselves, and those who can't won't get hung up by a question in their diagnostic logs. 
 
```suggestion
nvidia-smi || echo "no nvidia-smi, system either does not have an nvidia card or cuda is not properly configured"
```